__tests__/app.test.js

```diff
@@ -372,4 +371,20 @@ describe.only("DELETE /api/comments/:comment_id", () => {
         expect(body).toEqual({});
       });
   });
+  test("404: Responds with Not Found if comment_id is valid but does not exist in comments table", () => {
+    return request(app)
+      .delete("/api/comments/100000")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toEqual("Not Found");
+      });
+  })
+  test("400: Responds with Bad Request if comment_id is not an integer", () => {
```

Comment:
Again, great! Could just be really helpful with the message sent back to let the user know what has gone wrong.